### DISCUSSION TITLE
TV4 play

### DISCUSSION THREAD

**Discussion by 79Lindis** [stefansundin/privatkopiera#137] [General]
Är det fler som har problem med att ladda ner från tv4play?
![PP](https://user-images.githubusercontent.com/70215122/201925582-7abfb03e-2238-4b3c-a791-e45eae61b608.jpg)

**Comment by Per-Yve** [comment]
Har precis samma problem sedan några dagar tillbaka. Hittat någon lösning?

**Comment by zticky** [comment]
Sedan idag samma problem med svtplay.se

**Comment by stefansundin** [comment]
Det här ska nu vara fixat i v0.3.8. Förlåt att det dröjde. Läs mer här: https://github.com/stefansundin/privatkopiera/discussions/142
